添加文档 (#3)
dotnet-campus/dotnetCampus.CommandLineArgsViewer: README.md

# dotnetCampus.CommandLineArgsViewer

> This tool is used to observe that the parameters of the incoming command line are as expected.

Reviewer: You can view all the arguments passed from shells to this tool which helps you to debug command-lines.

要点：
* parameter 形参，argument 实参
* command line 单词应该加 `-`

| Build | NuGet |
|--|--|
|![](https://github.com/dotnet-campus/dotnetCampus.CommandLineArgsViewer/workflows/.NET%20Core/badge.svg)|[![](https://img.shields.io/nuget/v/dotnetCampus.CommandLineArgsViewer.svg)](https://www.nuget.org/packages/dotnetCampus.CommandLineArgsViewer)|

## Feature

> - Observe the count of the arguaments.

Reviewer: ```suggestion
- View the count of the arguments.
```

要点：
* Observe 观察，View 查看
* argument 单词拼写错误

> - The character that is escaped.

Reviewer: ```suggestion
- The characters which are escaped.
```

要点：
* 复数
* that 带从句，which 修饰定语

> - The difference between Kernal32 and .NET

Reviewer: ```suggestion
- View the command line loss from the shells (e.g. PowerShell lost quotes comparing to CMD).
- View the command line loss of the .NET version comparing with the Kernel32 version.
```

要点：
* Kernel32 拼写错误
* 同时说明了为什么要比较区别，比较的是哪一层的区别

## Install

```
dotnet tool install --global dotnetCampus.CommandLineArgsViewer
```

## Usage

> Input the arguments after ShowArgs, and you will find what is the application actually received arguments.

Reviewer: ```suggestion
Type any arguments after the command `ShowArgs`, and you'll view the command-line and args that an app (or a .NET app) will actually receive.
```

要点：
* 时态混乱

> ```
> ShowArgs [args]
> ```
> 

Reviewer: ````suggestion
```
ShowArgs [args]
```

For example, you can type the command line below:

```
ShowArgs /select,"D:\1,1.txt"
```

````

# dotnetCampus.CommandLineArgsViewer

> You can view all the arguments passed from shells to this tool which helps you to debug command-lines.

Reviewer: This is a tool to debug command-lines that you can view all the arguments passed from shells.

[中文](docs/zh-cn/README.md)

| Build | NuGet |
|--|--|
|![](https://github.com/dotnet-campus/dotnetCampus.CommandLineArgsViewer/workflows/.NET%20Core/badge.svg)|[![](https://img.shields.io/nuget/v/dotnetCampus.CommandLineArgsViewer.svg)](https://www.nuget.org/packages/dotnetCampus.CommandLineArgsViewer)|

## Feature

- View the count of the arguments.

> - The characters which are escaped.

Reviewer: View the shell escaping behavior of any characters (quote etc.)

> - View the command line loss of the .NET version comparing with the Kernel32 version.

Reviewer: 好像你少了一行，再看看上面提的问题？
